src/theory/bv/rewrites

```diff
@@ -594,22 +594,27 @@
   (def (power (int.log2 v)))
   (and (int.ispow2 v) (> v 1))
   (bvurem x (bv v n))
-  (concat (bv 0 (- n 1)) (extract (- power 1) 0 x)))
```

Comment:
What would happen if you merge this as is?

Reply:
I don't think anything, that's also why I approved it. But I was confused by this at first and it makes the PR hard to read, since one doesn't know what the new material is and what not, so I wanted to make you aware.